core/src/main/java/apoc/export/graphml/XmlGraphMLReader.java

```diff
@@ -277,16 +277,24 @@ public long parseXML(Reader input) throws XMLStreamException {
     private RelationshipType getRelationshipType(XMLEventReader reader) throws XMLStreamException {
         if (this.labels) {
             XMLEvent peek = reader.peek();
-            if (peek.isCharacters() && !(peek.asCharacters().isWhiteSpace())) {
+            boolean isChar = peek.isCharacters();
+            if (isChar && !(peek.asCharacters().isWhiteSpace())) {
```

Comment:
```suggestion
            boolean isWhiteSpace = peek.asCharacters().isWhiteSpace(
            if (isChar && !isWhiteSpace)) {
```

Reply:
La seconda condizione l'ho lasciata come prima perché altrimenti nel caso in cui `peek` non sia un `Character` dà un errore `Caused by: java.lang.ClassCastException: class com.sun.xml.internal.stream.events.StartElementEvent cannot be cast to class javax.xml.stream.events.Characters (com.sun.xml.internal.stream.events.StartElementEvent and javax.xml.stream.events.Characters are in module java.xml of loader 'bootstrap')`